.pipelines/.vsts-vhd-windows-automation.yaml

```diff
@@ -0,0 +1,27 @@
+pool:
+  # uncomment this if the auto teardown pool is down.
+  # vmImage: ubuntu-22.04
+  name: $(1ES_DevInfra_Auto_TearDown_Pool_Name)
+
+parameters:
+- name: ImageBump
+  displayName: Image Bump + Branch Cutting
+  type: boolean
+  default: true
+
+steps:
+- bash: |
+        az extension add -n azure-devops
+        echo $MAPPED_ADO_PAT | az devops login --organization=https://dev.azure.com/msazure
+        az devops configure --defaults organization=https://dev.azure.com/msazure project=CloudNativeCompute
+  env:
+        MAPPED_ADO_PAT: $(ADO_PAT)
+  displayName: 'az devops login'
+- bash: |
+        echo "Bumping windows VHD base image version"
+        /bin/bash vhdbuilder/scripts/windows/automate_version_bump.sh $MAPPED_GITHUB_PAT $GITHUB_USER_NAME $CHERRY_PICK_COMMIT_ID
+  env:
+        MAPPED_GITHUB_PAT: $(GITHUB_PAT)
```

Comment:
Could we use parameters instead of variables since we always need to set them in every build?

Reply:
It's set in the pipeline variable with a default value. We only need to set it if we want to override the default value. It is stored as a secret. We don't need to click "Add variable" in each build.

Reply:
I think that we need the owner to set those parameters/varialbes since the owner needs to update the PR, for example, updating the registries, after the pr is created.
Keeping GITHUB_PAT as secret variable is good.

Reply:
The default value of GITHUB_PAT is now set to an empty string and the owner needs to set this explicitly during each run. Does this look good?